app/code/core/Mage/Core/Model/Session/Abstract/Varien.php

```diff
@@ -32,7 +32,8 @@ class Mage_Core_Model_Session_Abstract_Varien extends Varien_Object
     const VALIDATOR_HTTP_X_FORVARDED_FOR_KEY    = 'http_x_forwarded_for';
     const VALIDATOR_HTTP_VIA_KEY                = 'http_via';
     const VALIDATOR_REMOTE_ADDR_KEY             = 'remote_addr';
-    const VALIDATOR_SESSION_EXPIRE_TIMESTAMP    = 'session_expire_timestamp';
```

Comment:
removing this could cause errors in modules that use this global variable. I would refrain form removing core variables.

Reply:
I agree here. no breaking changes please

Reply:
Added back.